TechtonicaModLoader/App.xaml.cs

```diff
@@ -16,9 +15,10 @@ public partial class App : Application
 
         private Log? logger = null;
         private MainViewModel? mainVeiwModel = null;
-        private ProfileManager? profileManager = null;
         private DialogService? dialogService = null;
 
+        private readonly IProfileManager _profileManager = new ProfileManager();
```

Comment:
This will be the one and only IProfileManager instance.

Reply:
Yes.   To clarify, for code running via App.xaml.cs, that will be the one instance.